queue/queue.go

```diff
@@ -111,7 +111,8 @@ func (qm *Manager) declareQueue() error {
 // ConsumeMessages is used to consume messages that are sent to the queue
 // Question, do we really want to ack messages that fail to be processed?
 // Perhaps the error was temporary, and we allow it to be retried?
-func (qm *Manager) ConsumeMessages(ctx context.Context, wg *sync.WaitGroup, db *gorm.DB, cfg *config.TemporalConfig) error {
+func (qm *Manager) ConsumeMessages(ctx context.Context, wg *sync.WaitGroup, db *gorm.DB, cfg *config.TemporalConfig, dev bool) error {
```

Comment:
why is this per-`ConsumeMessages`? why cant it be set as part of `Manager`?

Reply:
good point what would be simpler